src/main/java/com/example/controller/GenerateGraphController.java

```diff
@@ -59,5 +72,20 @@ protected void updateItem(DefinedGraph definedGraph, boolean empty) {
         ObservableList<DefinedGraph> definedGraphs = FXCollections.observableArrayList(DefinedGraph.FULL);
         graphBox.setItems(definedGraphs);
         graphBox.getSelectionModel().select(0);
+
+        vertices.setContainedSetting(new AlgorithmSetting<>
+                ("vertices", (int) verticesSlider.getValue(), Integer.class,
+                        (value) -> value >= verticesSlider.getMin() && value <= verticesSlider.getMax()));
+
+        verticesSlider.valueProperty().addListener((observable, oldValue, newValue) -> {
```

Comment:
Did you try using bidirectional binding? Seems to fit perfectly here

Reply:
There is a problem with verticesSlider's DoubleProperty using bidirectional binding. It's hard to get int value this way.